bitgo-java-api/build.gradle

```diff
@@ -1,14 +1,17 @@
 apply plugin:'java'
 apply plugin:'idea'
-version='0.0.4'
+
+group = 'com.bitso'
+version='SNAPSHOT'
 
 repositories {
     mavenCentral()
+    maven { url 'https://jitpack.io' }
 }
 
 dependencies {
-    compileOnly 'org.projectlombok:lombok:1.16.16'
-    compile 'org.slf4j:slf4j-api:1.7.25',
-            'com.alibaba:fastjson:1.2.31'
+    compile 'org.projectlombok:lombok:1.+'
+    compile 'org.slf4j:slf4j-api:1.7.+',
+            'com.fasterxml.jackson.core:jackson-databind:2.+'
```

Comment:
You should fix the version (even if they need updating)

Reply:
Done, checked in